src/main/java/teammates/ui/template/StudentProfileEditBox.java

```diff
@@ -0,0 +1,73 @@
+package teammates.ui.template;
+
+import teammates.common.util.StringHelper;
+
+public class StudentProfileEditBox {
+
+    private String name;
+    private String editPhoto;
+    private String shortName;
+    private String email;
+    private String institute;
+    private String nationality;
+    private String gender;
+    private String moreInfo;
+    private String googleId;
+    private String pictureUrl;
+
+    public StudentProfileEditBox(String name, String editPhoto, String shortName, String email,
+                                 String institute, String nationality, String gender,
+                                 String moreInfo, String googleId, String pictureUrl) {
+        this.name = name;
+        this.editPhoto = editPhoto;
```

Comment:
Should this be `isEditingPhoto` as well to match with the *PageData?
On a side note, from what I recall, `${thisIsABoolean}` will result in a "true" or "false" string when outside of a `if` condition in the .tag.
It may be cleaner to store it as a boolean, but it's fine either way; shouldn't be any issue.